tdrs-backend/setup.cfg

```diff
@@ -50,7 +50,7 @@ skip_gitignore = True
 use_parentheses = True
 
 [tool:pytest]
-DJANGO_SETTINGS_MODULE = tdpservice.settings.local
+DJANGO_SETTINGS_MODULE = tdpservice.settings.test
```

Comment:
This is just an incidental fix-up, right?

Reply:
Yeah I noticed that there was a separate settings module set up for Tests, I utilized it to provide a default value for the `BASE64_DECODE_JWT_KEY` variable which is a boolean that can be used to allow defining the `JWT_KEY` environment variable as a base 64 encoded string. This plays much nicer with docker-compose's interpretation of `env_file` for local instances but shouldn't be used during tests so that settings module will set it to False.

I did find that the environment variable `DJANGO_SETTINGS_MODULE` passed in to the container overshadows this configuration since tests are run in the same container as the backend server. As such, I'll update the test command in the README to utilize this module when calling pytest.

Reply:
I'm actually gonna revert this change, I found an alternate way that doesn't require the use of the `BASE64_DECODE_JWT_KEY` setting and this line in particular gets overshadowed by the docker environment so its an inconsequential change.